nextmove/src/main/java/no/difi/meldingsutveksling/nextmove/DpiConversationResource.java

```diff
@@ -16,4 +19,27 @@
 
     private String title;
 
+    private DpiConversationResource(String conversationId,
+                                    String senderId,
+                                    String senderName,
+                                    String receiverId,
+                                    String receiverName,
+                                    LocalDateTime lastUpdate,
+                                    Map fileRefs,
+                                    Map customProperties) {
+        super(conversationId, senderId, senderName, receiverId, receiverName, ServiceIdentifier.DPI, lastUpdate, fileRefs, customProperties);
+    }
+
+    public static DpiConversationResource of(ConversationResource cr) {
+        DpiConversationResource dpi = new DpiConversationResource(cr.getConversationId(),
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "dpi". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1488)